src/Halogen/VDom/Driver.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+exports.createElementFromHTML = function(html) {
+  // used: https://stackoverflow.com/a/35385518/1833322
+  // alternatively: https://stackoverflow.com/a/494348/1833322
+  var template = document.createElement('template');
+  html = html.trim(); // Never return a text node of whitespace as the result
+  template.innerHTML = html;
+  var newElement = template.content.firstChild;
```

Comment:
Does this mean it only supports a single HTML element? That should at least be documented somewhere. Maybe with support for fragments it could be improved to capture all children.

Reply:
I hadn't considered that .. but good point. I guess it does mean that.

It doesn't necessarily need to be like this but inserting multiple elements would need to happen at the parent of the current element. Which i think would make the code unnecessary more complicated and might not even be possible (what to do if the raw-html was the top halogen element).

Also i think it's unwanted to enter multiple nodes in one location. Because you can just put a node around it. i.e. `<b>node1</b><b>node2</b>` into `<div><b>node1</b><b>node2</b></div>`.

What could be done is count the amount of children and if it's more than 1 to show a [console warning](https://developer.mozilla.org/en-US/docs/Web/API/Console/warn) on runtime.

Yes and documentation.